certified-connectors/Tendocs Documents/apiDefinition.swagger.json

```diff
@@ -108,9 +108,243 @@
             "description": "Something has gone wrong, please contact support@tendocs.com"
           }
         },
-        "summary": "Convert Document To New File Format",
+        "summary": "Convert Document To New File Format (PDF/HTML/TXT)",
         "description": "Convert a document to a new file format. Includes PDF and HTML."
       }
+    },
+    "/ai/v1/tasks/summary": {
+      "post": {
+        "tags": [
+          "Documents",
+          "AI",
+          "Summary"
+        ],
+        "operationId": "V1AiSummaryPost",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AiSummaryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "403": {
+            "description": "The request is not authorized to make the conversion request"
+          },
+          "404": {
+            "description": "Service has not been found, please contact support@tendocs.com"
+          },
+          "500": {
+            "description": "Something has gone wrong, please contact support@tendocs.com"
+          },
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Type": {
+                  "type": "string"
+                },
+                "Content": {
+                  "type": "string"
+                },
+                "Length": {
+                  "type": "number"
+                }
+              },
+              "required": [
+                "Type",
+                "Content",
+                "Length"
+              ]
+            }
+          }
+        },
+        "summary": "Summarise a document",
+        "description": "Create a AI (GPT-4) generated summary of the provided document."
+      }
+    },
+    "/ai/v1/tasks/summary/result": {
+      "get": {
```

Comment:
Kindly fix the below validation error. 

##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1ai~1v1~1tasks~1summary~1result/get/responses/default